### ISSUE TITLE
Doc: Fix typo in the doc page about the tests

### ISSUE THREAD

**Issue by ChristopherAndre** [zio/zio#2375]
Doing a copy paste from [the "Testing" page](https://zio.dev/docs/usecases/usecases_testing) made me realize there seems to be a typo in the part about the SBT imports.

Indeed, we have:
```
testFrameworks += Seq(new TestFramework("zio.test.sbt.ZTestFramework"))
```
But it looks like sbt is expecting something like either
```
testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework")
```
Or, if we prefer to keep the `Seq`:
```
testFrameworks ++= Seq(new TestFramework("zio.test.sbt.ZTestFramework"))
```

**Comment by adamgfraser** [comment]
@ChristopherAndre You are completely right. We actually have an open PR regarding this at #2284. Let me check with the author and see if we can get that over the finish line. Thanks for raising!

**Comment by ChristopherAndre** [comment]
Thanks for the reply, @adamgfraser : I was about to submit a PR for this!

**Comment by ghostdogpr** [comment]
Closed by #2284
